.github/scripts/build.sh

```diff
@@ -64,3 +68,12 @@ echo =====
 ${DOCKER} bash -c "cd build && cpack -G TGZ -D CPACK_PACKAGE_FILE_NAME=psidk-ubuntu-${UBUNTU_BUILD_VER}"
 echo =====
 ${DOCKER} bash -c "cd build && mv book psidk-book && tar czf ../psidk-book.tar.gz psidk-book"
+echo =====1
+${DOCKER} bash -c "cargo generate -p ./package-templates/ --destination ./services/user/ --init -v --name Buildtest --silent basic-01"
+echo =====2
+${DOCKER} bash -c "ls -la /home/runner/work/psibase/psibase"
+${DOCKER} bash -c "ls -la /home/runner/work/psibase/psibase/build/rust/release"
+echo =====3
+# ${DOCKER} bash -c "chmod -R 777 /root/psibase/build/rust/release/cargo-psibase"
+# echo =====4
+${DOCKER} bash -c "cd services/user/Buildtest && /home/runner/work/psibase/psibase/build/rust/release/cargo-psibase package"
```

Comment:
You should use `${WORKSPACE_ROOT}` instead of `/home/runner/work/psibase/psibase`

Reply:
ah, ok.